packages/windows/vcruntime/vs2017/readme.txt

```diff
@@ -0,0 +1,17 @@
+Subject to Microsoft Visual Studio Community 2017 EULA [1], This package
+ contains Visual C++ Runtime files as redistributable code:
+
+[1] https://visualstudio.microsoft.com/license-terms/mlt553321/

```

Comment:
This link connects to the Japanese page. At least, in my environment.

Reply:
It also depends on language navigation setting on bottom of website

Reply:
Oh.
Thank you for your information.
I could display the English page when I changed the language setting to "English".